files/routes/feeds.py

```diff
@@ -21,34 +22,36 @@ def feeds_user(v=None, sort='hot', t='all'):
 		t=t,
 		v=None,
 		)
-	
 	posts = get_posts(ids)
 
-	domain = environ.get("DOMAIN").strip()
-
 	doc, tag, text = Doc().tagtext()
-
 	with tag("feed", ("xmlns:media","http://search.yahoo.com/mrss/"), xmlns="http://www.w3.org/2005/Atom",):
 		with tag("title", type="text"):
-			text(f"{sort} posts from {domain}")
+			text(f"{sort} posts from {SITE}")
```

Comment:
We don't actually have SITE anymore :V We've got SITE_ID and SITE_TITLE; in this case you want SITE_TITLE, which is the human-readable version.

Reply:
FWIW, there is still a SITE lurking around, which contains the `DOMAIN` env variable (which is what it was referencing there before the change):
https://github.com/themotte/rDrama/blob/e12b0eea1a6bdfbf6981e169b0aefcc634f81a5c/files/helpers/const.py#L10

It is confusing, though, and I'll switch it over to SITE_TITLE — it's also more human-readable than the domain name.